### DISCUSSION TITLE
SageTV taking a loooooooooong time to initialize when booting in Windows 10 - thoughts?

### DISCUSSION THREAD

**Discussion by Sagetvmaster** [OpenSageTV/Community#146] [General]
When I looked in the SageTV directory, I found over 300 Sage.properties.backup files, all with unique identifiers.
I moved them to a useless dir, and tried rebooting - and it seemed to be much quicker.
Any reason why that MAY be the problem?
Anyone else seeing very long initialization times when first starting SageTV in Windows, I mean like five minutes?
Yes, 'limited_carny=true' is properly set to speed things up in Sage.properties
thanks

**Comment by Sagetvmaster** [comment]
Thought the problem was a damaged drive.  Replaced that drive - but still getting hit with incredible long initialization and startup times.  After the SageTV server boots up, it often goes into an equally long 'Updating' phase.
Anyone? No one?  It just started like this a month or so ago, so it's a new problem for sure
All drives  check out using Speccy

**Reply by trk2** [reply]
The SageTV service takes a long time?  Or the client UI on the server?

**Reply by Sagetvmaster** [reply]
Server, with the service running in the background

**Comment by pendragonsound** [comment]
I've seen SageTV service take a longer time when it gets 'stuck' on media files that it can't figure out. Removing those files always sped it up. On the other hand, having a lot of media files can take a long time to scan, especially if they're on another LAN machine. Currently I have our SageTV service running on a pretty fast Linux machine. With ~20,000 media files served from other servers,, a complete startup scan can still take 10-15 minutes. But that's better than when I had SageTV running on Windows, when it could easily take 30-40 minutes starting with the identical file servers..

**Reply by Sagetvmaster** [reply]
I think there may indeed be either a bad file, or a checksum error on one of the drives.  I've replaced that drive and am now running tests.  But yes, my library, like most of us who've been here for nearly 20 years, is pretty extensive...

**Reply by wayner9** [reply]
When SageTV scans media files it scans all files, including music and picture files.  If you have a large media library then this adds tens of thousands of files to scan which hugely slows down the scanning process.  Same thing with photos.  So I no longer have music and photos added to SageTV.  I had requested the developers to allow you to be able to add a function to scan just video files but that was never implemented.

Years ago I had some image files in tiff format that caused SageTV to crash when the Screensaving was trying to display those files.

**Reply by Sagetvmaster** [reply]
Turns out one of my two, major recording drives had started to go bad, and even with attempting to fix checksum errors, it was time for replacement.  Luckily I got all but three recordings out of 2gb of data moved to the new drive.  Things have been booting much quicker.  Thanks for the help
